### PR TITLE
Add utility methods Script#color?, Script#color_id, Script#remove_color

### PR DESCRIPTION
**Pull request by Yamaguchi**
Add methods to Tapyrus::Script
- Tapyrus::Script#color?
- Tapyrus::Script#color_id
- Tapyrus::Script#remove_color

### CODE REVIEW

File: lib/tapyrus/script/script.rb [hunk 1]
```diff
@@ -230,6 +242,16 @@ def cp2sh?
       return false unless chunks[1].ord == OP_COLOR
       OP_HASH160 == chunks[2].ord && OP_EQUAL == chunks[4].ord && chunks[3].bytesize == 21
     end
+
```
Comment (azuchi) [inline lib/tapyrus/script/script.rb:245]: Please add method description.

File: lib/tapyrus/script/script.rb [hunk 2]
```diff
@@ -230,6 +242,16 @@ def cp2sh?
       return false unless chunks[1].ord == OP_COLOR
       OP_HASH160 == chunks[2].ord && OP_EQUAL == chunks[4].ord && chunks[3].bytesize == 21
     end
+  
+    def colored?
+      cp2pkh? || cp2sh?
+    end
+
```
Comment (azuchi) [inline lib/tapyrus/script/script.rb:249]: Please add method description.

File: lib/tapyrus/script/script.rb (hunk 1 above)
Comment (Yamaguchi) [reply lib/tapyrus/script/script.rb:249]: Fixed at 090d465

File: lib/tapyrus/script/script.rb (hunk 2 above)
Comment (Yamaguchi) [reply lib/tapyrus/script/script.rb:255]: Fixed at 090d465

**Pull request merged**